src/utils/vc/credentials.js

```diff
@@ -170,6 +173,9 @@ export async function verifyCredential(credential, {
     throw new Error('Passing resolver and documentLoader results in resolver being ignored, please re-factor.');
   }
 
+  const isJWT = typeof vcJSONorString === 'string';
+  const credential = isJWT ? JSON.parse(base64url.decode(vcJSONorString.split('.')[1])).vc : vcJSONorString;
```

Comment:
are these array indices guaranteed? (same for the one on line 224)

Reply:
if it's a valid JWT format then yeah they are